CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.7.0-rc.0 / 2019-01-18
+
+This release adds support for PromQL Subquries and disk size based retention. 
```

Comment:
We only include a preamble where there's a significant change, which none of these are. The disk warning should go on the appropriate line

Reply:
Hmm, I think both subqueries and an highly anticipated _experimental_ change should be highlighted. Atleast that it's experimental.

Reply:
I'd expect only big breaking stuff to be mentioned, I wouldn't want to fatigue users so they miss one of those.